### PR TITLE
Add June 14 virtual mesh event

### PR DESCRIPTION
**Pull request by dasanchez**

### CODE REVIEW

File: _posts/2018-06-14-virtual-mesh.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+location: "#virtualmesh:tomesh.net"
+locationLink: https://chat.tomesh.net/#/room/#virtualmesh:tomesh.net
+date: 2018-06-14
+startTime: '19:30'
+endTime: '21:30'
+---
+
+Some Toronto Mesh members are connected through a virtual mesh network. This is a transitional phase where our mesh nodes are connected to one another using an internet connection instead of a physical line or radio. We are nonetheless able to test out distributed applications as if we were connected through a physical mesh.
+
+This month we are meeting virtually on [#virtualmesh:tomesh.net](https://chat.tomesh.net/#/room/#virtualmesh:tomesh.net) to do some hands-on hacking. We will:
```
Comment (benhylau) [inline _posts/2018-06-14-virtual-mesh.md:15]: This month we are also meeting actually at Install Night :)

File: _posts/2018-06-14-virtual-mesh.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Set up a [Secure Scuttlebutt](https://www.scuttlebutt.nz/) network
+- Run streaming tests over [IPFS](https://ipfs.io)
```
Comment (benhylau) [inline _posts/2018-06-14-virtual-mesh.md:18]: Something came up for me and I will miss this virtual mesh. Will @darkdrgn2k and @ASoTNetworks be able to facilitate this? I am probably able to help via phone only.

**Review by benhylau — APPROVED** [review]

File: _posts/2018-06-14-virtual-mesh.md (hunk 2 above)
Comment (benhylau) [reply _posts/2018-06-14-virtual-mesh.md:18]: Assigned @darkdrgn2k and @ASoTNetworks 

If you guys can take care of the SSB pub and IPFS testing please go ahead and merge.

**Comment by darkdrgn2k** [comment]
I'll be there

**Comment by ASoTNetworks** [comment]
I will be online too.

**Pull request merged**